variables.tf

```diff
@@ -1,11 +1,8 @@
 # Network Security Group definition
 variable "resource_group_name" {
-  default     = "nsg_rg"
```

Comment:
The reason of removing this default value is?

Reply:
Before, by default, it'll create a resource group. Now we import an existing resource group. So there is no default name for an existing resource group.